Makefile

```diff
@@ -0,0 +1,123 @@
+## Haiku Generic Makefile v2.6 ##
+
+## Fill in this file to specify the project being created, and the referenced
+## Makefile-Engine will do all of the hard work for you. This handles any
+## architecture of Haiku.
+
+# The name of the binary.
+NAME = BeMobile
+
+# The type of binary, must be one of:
+#	APP:	Application
+#	SHARED:	Shared library or add-on
+#	STATIC:	Static library archive
+#	DRIVER: Kernel driver
+TYPE = APP
+
+# 	If you plan to use localization, specify the application's MIME signature.
+APP_MIME_SIG = 
+
+#	The following lines tell Pe and Eddie where the SRCS, RDEFS, and RSRCS are
+#	so that Pe and Eddie can fill them in for you.
+#%{
+# @src->@ 
+
+#	Specify the source files to use. Full paths or paths relative to the 
+#	Makefile can be included. All files, regardless of directory, will have
+#	their object files created in the common object directory. Note that this
+#	means this Makefile will not work correctly if two source files with the
+#	same name (source.c or source.cpp) are included from different directories.
+#	Also note that spaces in folder names do not work well with this Makefile.
+SRCS =  Pattern.cpp Matcher.cpp santa/BetterScrollView/BetterScrollView.cpp santa/ColumnListView/CLVColumnLabelView.cpp santa/ColumnListView/CLVEasyItem.cpp santa/ColumnListView/CLVListItem.cpp santa/ColumnListView/ColumnListView.cpp santa/Cursors/Cursors.cpp santa/NewStrings/NewStrings.cpp santa/PrefilledBitmap/PrefilledBitmap.cpp santa/ScrollViewCorner/ScrollViewCorner.cpp phonelist.cpp callregslotview.cpp phonelistslotview.cpp bemobile.cpp dialabout.cpp dialeditpbook.cpp dialnewsms.cpp dialcalevent.cpp dialcalendar.cpp gsm.cpp main.cpp mainwindow.cpp workview.cpp calview.cpp pbbynameview.cpp pbview.cpp smsview.cpp smsboxview.cpp smsinboxview.cpp statusview.cpp mobileview.cpp initwindow.cpp
```

Comment:
This would be nicer when in alphabetical order (maybe all starting with "A" on one line, the "B" and so on. Also respect the 80 char limit by putting a ` \` and a line break at the end of every line.